src/unstract/adapters/vectordb/postgres/src/postgres.py

```diff
@@ -1,4 +1,5 @@
 import os
+import urllib.parse
```

Comment:
```suggestion
from urllib.parse import quote_plus
```
@harini-venkataraman Please fix the import and use it below.

Fix the same in Supase too.